src/app.js

```diff
@@ -274,6 +261,44 @@ class SAFEApp extends EventEmitter {
     return this._appInfo;
   }
 
+  get CONSTANTS() { // eslint-disable-line class-methods-use-this
```

Comment:
We don't need to expose this here, we should just expose them from the global CONSTANTS object we are already exposing from here: https://github.com/maidsafe/safe_app_nodejs/blob/master/src/index.js#L4